clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/HttpCloudFoundryClient.java

```diff
@@ -106,9 +109,12 @@ public HttpCloudFoundryClient(
     okHttpClient =
         okHttpClient
             .newBuilder()
+            .connectTimeout(clientConfig.getConnectionTimeout(), TimeUnit.MILLISECONDS)
```

Comment:
Looks like we are setting these twice. Once in CloudFoundryProviderConfig for the bean cloudFoundryOkHttpClient and then here as well. I don't think I have a preference as to which one we use

Reply:
Removed.